Kontent.Ai.Management/Models/VariantFilter/VariantFilterFiltersModel.cs

```diff
@@ -0,0 +1,59 @@
+using Kontent.Ai.Management.Models.Shared;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Kontent.Ai.Management.Models.VariantFilter;
+
+/// <summary>
+/// Represents the variant filter filters model.
+/// </summary>
+public class VariantFilterFiltersModel
+{
+    /// <summary>
+    /// Gets or sets the search phrase.
+    /// </summary>
+    [JsonProperty("search_phrase")]
+    public string SearchPhrase { get; set; }
+
+    /// <summary>
+    /// Gets or sets the language.
+    /// </summary>
+    [JsonProperty("language")]
+    public Reference Language { get; set; }
+
+    /// <summary>
+    /// Gets or sets the content types.
+    /// </summary>
+    [JsonProperty("content_types")]
+    public IEnumerable<Reference> ContentTypes { get; set; }
+
+    /// <summary>
+    /// Gets or sets the contributors.
+    /// </summary>
+    [JsonProperty("contributors")]
+    public IEnumerable<UserIdentifier> Contributors { get; set; }
+
+    /// <summary>
+    /// Gets or sets whether to filter items with no contributors.
+    /// </summary>
+    [JsonProperty("has_no_contributors")]
+    public bool? HasNoContributors { get; set; }
+
+    /// <summary>
+    /// Gets or sets the completion statuses.
+    /// </summary>
+    [JsonProperty("completion_statuses")]
+    public IEnumerable<string> CompletionStatuses { get; set; }
```

Comment:
Not sure if we have any other places like this but an enum seems much more user-friendly.